teletext.go

```diff
@@ -859,7 +859,7 @@ func parseTeletextRow(i *Item, d decoder, fs func() styler, row []byte) {
 	// Loop through columns
 	var l = Line{}
 	var li = LineItem{InlineStyle: &StyleAttributes{}}
-	var started bool
+
```

Comment:
This part is really important for teletext subtitles.

Why do you need it removed ?

Reply:
We have some STL files that they do not have startbox (0x0B) flag and endbox flag (0x0A)

Reply:
Some old files. Nowadays I think taht we do not have files like these. Therefore we could leave this part

Reply:
Can you revert this line as well as the `if started` you removed below ?

Reply:
It is done in the last commit